README.md

```diff
@@ -11,13 +11,13 @@ Since the [cipd executable](https://chromium.googlesource.com/infra/luci/luci-go
 currently used to update V8 and its dependencies, and apply the z/OS port, after
 the initial [fetch v8](https://v8.dev/docs/source-code) (also included below with a comment).
 
-This workflow requires two platforms.  One is USS which is the target platform.  The second is a generic linux
+This workflow requires two platforms.  One is z/OS® UNIX System Services (z/OS UNIX) which is the target platform.  The second is a generic linux
 AMD64 platform to get files normally acquired via `cipd`
 
 
-#### Linux AMD64 plaform setup
+#### Linux plaform (non z/OS® UNIX System Services (z/OS UNIX)) setup
 
-This is the workflow on the non USS platform for obtaining files via `cipd`.  Do
+This is the workflow on the non z/OS® UNIX System Services (z/OS UNIX) platform for obtaining files via `cipd`.  Do
```

Comment:
Suggest replacing:
This is the workflow on the non z/OS® UNIX System Services (z/OS UNIX) platform
by:
This is the workflow on the non-z/OS UNIX platform

since "z/OS® UNIX System Services (z/OS UNIX)" was already specified above.